tsconfig.json

```diff
@@ -4,7 +4,7 @@
 
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "ESNext",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Denne måtte endres for å få javascript ting til å finne typescript ting på server.
Aner egentlig ikke konsekvensene av det så gjerne noen kommentarer her :sweat_smile:

Reply:
Nye verdien er stjelt av razzle eksempelet her
https://github.com/jaredpalmer/razzle/blob/master/examples/with-typescript/tsconfig.json